session/bootstrap_serial_test.go

```diff
@@ -827,6 +827,45 @@ func TestUpgradeVersion75(t *testing.T) {
 	require.Equal(t, "char(255)", strings.ToLower(row.GetString(1)))
 }
 
+func TestUpgradeVersion80(t *testing.T) {
```

Comment:
How about Implementing this test in this way: 1) upgrade to version 79; 2) check and ensure `stats_history` and `stats_meta_history` are not existed; 3) upgrade to the version 80; 4) check and ensure these two tables are existed;

Reply:
Okk. I considered that the `stats_history` and `stats_meta_history` could be created manually at version 79. That could affect the upgrade progress so I dropped it if existed. 

It seems that checking it before upgrade is more reasonable. I will take it.